terraform/provider_incus.tf

```diff
@@ -0,0 +1,13 @@
+provider "incus" {
+  # Incus client library looks in ~/.config/incus for client.crt and client.key for authentication
+  generate_client_certificates = true
+  accept_remote_certificate    = true
+
+  remote {
+    name    = var.incus_host_name
+    scheme  = var.incus_host_scheme
+    address = var.incus_host_address
+    port    = var.incus_host_port
+    default = var.incus_host_default
```

Comment:
This isn't an actual config key for the remote definition in the provider as far as I can tell.
It's a config key in the Incus config.yml but that's not something that's useful in this context.

I'll drop that one when pushing through my branch.

Reply:
Actually, it looks like it's used as a way to tell the Terraform provider which remote to use by default (when not mentioned through `incus_remote`), so it's then likely preferable to just always set it to `true` in this context.

Reply:
At which point we can also fix its name to something we control to reduce the number of variables, say `incus-deploy` in this case.